FYI: fix printing of negative space change
http://sourceware.org/ml/gdb-patches/2011-04/msg00017.html
http://sourceware.org/ml/gdb-cvs/2011-04/msg00009.html

### src/gdb/ChangeLog	2011/04/01 17:03:23	1.12891
### src/gdb/ChangeLog	2011/04/01 17:20:37	1.12892
## -1,3 +1,8 @@
+2011-04-01  Tom Tromey  <tromey@redhat.com>
+
+	* utils.c (report_command_stats): Don't print `-' for negative
+	number.
+
 2011-04-01  Eric Botcazou  <ebotcazou@adacore.com>
 
 	* ada-lang.c (ada_is_simple_array_type, ada_value_slice_from_ptr)
--- src/gdb/utils.c	2011/03/31 14:32:48	1.250
+++ src/gdb/utils.c	2011/04/01 17:20:42	1.251
@@ -657,10 +657,10 @@
       long space_diff = space_now - start_stats->start_space;
 
       printf_unfiltered (msg_type == 0
-			 ? _("Space used: %ld (%c%ld during startup)\n")
-			 : _("Space used: %ld (%c%ld for this command)\n"),
+			 ? _("Space used: %ld (%s%ld during startup)\n")
+			 : _("Space used: %ld (%s%ld for this command)\n"),
 			 space_now,
-			 (space_diff >= 0 ? '+' : '-'),
+			 (space_diff >= 0 ? "+" : ""),
 			 space_diff);
 #endif
     }
